concrete/src/Feature/UsesFeatureInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Concrete\Core\Feature;
+
+interface UsesFeatureInterface
+{
+
+    /**
+     * @return string[] Feature handles
```

Comment:
Adding a description to this and an example of output: `return [Feature::SOME_FEATURE]` would make it easier to implement without needing to reference and existing implementation